### PR TITLE
forklift pallet

### LINKED ISSUE #1: Offender Data for Application Development
### Summary

- [x] Obtain geocoded offender data from agrc sftp
  - data are updated each morning by about 7 a.m.
- [ ] Obtain lookup data from [drive sheet](https://docs.google.com/spreadsheets/d/1qE7BFMH5xTmvQ9-H8Yo16SwZNxAqBAlgRaG1HFIIbAU/edit#gid=1065322794)
- [x] Add forklift process to pull data each day (even though dev data may not change through application development)

### PR DESCRIPTION
**Pull request by steveoh**
this will currently write the dat data to a sql table if the modified time of the drop off file is newer than what we keep on disk. 

The pallet truncates the data in the file but persists the last modified times.

The table is truncated on every run.

A geography field is created now using the lat lon values.

close #1

### CODE REVIEW

**Comment by stdavis** [comment]
This looks good to me.

**Comment by steveoh** [comment]
@stdavis will you review this again?

File: scripts/schema.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    legal_status_code nchar(1),
+    legal_status_description nvarchar(30),
+    supervision_start_date datetime,
+    offender_location nvarchar(40),
+    address_start_date datetime,
+    address nvarchar(50),
+    unit nvarchar(15),
+    city nvarchar(30),
+    state nchar(2),
+    zip integer(5),
```
Comment (stdavis) [inline scripts/schema.sql]: Should this be `zip integer`? I got "Cannot specify a column width on data type int" when I tried to run this script.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "python.pythonPath": ".env\\Scripts\\python.exe",
```
Comment (stdavis) [inline .vscode/settings.json]: Mine ended up being `"python.pythonPath": ".env/bin/python",`. I used python3 installed via Homebrew. What did I do wrong?

File: forklift/pallet.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        finally:
+            with open(dat_file, 'w') as dat:
+                dat.truncate()
+
+            if success:
+                #: reset access stats from truncate
+                os.utime(dat_file, (stats.st_atime, stats.st_mtime))
+            else:
+                #: reset stats to original because errors
+                os.utime(dat_file, (self.data_stats.st_atime, stats.st_mtime))
```
Comment (stdavis) [inline forklift/pallet.py]: This doesn't work if the pallet fails on its first run. Not sure how to handle that but maybe something to think about. Maybe check to see if the two modified times are the same and if so then delete the file?

File: .vscode/settings.json (hunk 2 above)
Comment (steveoh) [reply .vscode/settings.json]: windows/mac difference...

File: forklift/pallet.py (hunk 3 above)
Comment (steveoh) [reply forklift/pallet.py]: i think my update will fix this. i'll request a re review...

**Pull request merged**